css/src/components/buttons.css

```diff
@@ -0,0 +1,88 @@
+.yoast .yoast-button {
+	display: inline-block;
+	position: relative;
+	padding: 0.7em 1.1em 0.9em;
+	text-decoration: none;
+	border-radius: 0.3em;
+	border: 1px solid rgba(0,0,0, 0.2);
```

Comment:
the opacity in the border can be .2 instead of 0.2. I dont mind what we do, as long we are on the same page.